dev-proxy-abstractions/ILogger.cs

```diff
@@ -40,15 +41,20 @@ public enum LogLevel
     Error
 }
 
-public interface ILogger : ICloneable
+/// <summary>
+/// Custom interface for logging, extending <see cref="Microsoft.Extensions.Logging.ILogger"/>
+/// </summary>
+/// <remarks>
+/// Please use structured logging as much as possible.
```

Comment:
Let's remove these remarks. Since we're the sole user of the interface, we don't need them.

Reply:
What about others building plugins?

Reply:
We're changing the interface in a backwards incompatible way which will require anyone who built a plugin in the past to adapt. That said: the new way of logging is the only way we're exposing and our plugins, and in the future docs, will give enough guidance.